### ISSUE TITLE
sonoff micro usb - status not updating (Cannot read property 'updateCharacteristic' of undefined)

### ISSUE THREAD

**Issue by daiiig** [bwp91/homebridge-ewelink#178]
### What issue do you have? Please be as thorough and explicit as possible.
just installed this plugin today (1/4/20201) and added a sonoff micro usb plug and am getting an error (Cannot read property 'updateCharacteristic' of undefined.) whenever the device changes state outside of the homekit/homebridge (app, physical button, etc).  Status update works fine if turned on/off through homekit/homebridge.

### Details of your setup.
* Do you use (1) Homebridge (+ config-ui), (2) Homebridge (CLI) or (3) HOOBS? 

Homebridge (+ config-ui)

* Which version of Homebridge/HOOBS do you have?
Homebridge
v1.1.7

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.

OS-Raspbian GNU/Linux Buster (10)
Node.js Version-v14.15.3
Npm Version-v6.14.9

* Which version of this plugin (homebridge-ewelink) do you have? Has the issue started since upgrading from a previous version?
just installed this plugin today (1/4/2021) 
homebridge-ewelink v4.0.2

* Which eWeLink devices do you have that are causing issues? Please include product models if applicable.

Brand	SONOFF
Model (UIID)	Micro (77) - version 3.4.0
Shared Device	No

### Please paste any relevant logs below.

```
[04/01/2021, 17:53:12] [eWeLink] [btm.tv.usb] receiving update {"switches":[{"switch":"on","outlet":0},{"switch":"off","outlet":1},{"switch":"off","outlet":2},{"switch":"off","outlet":3}],"online":true,"updateSource":"WS"}.
[04/01/2021, 17:53:12] [eWeLink] [btm.tv.usb] sending update failed as Cannot read property 'updateCharacteristic' of undefined.
[04/01/2021, 17:53:12] [eWeLink] [btm.tv.usb] receiving update {"switches":[{"switch":"on","outlet":0},{"switch":"off","outlet":1},{"switch":"off","outlet":2},{"switch":"off","outlet":3}],"online":true,"updateSource":"LAN"}.
[04/01/2021, 17:53:12] [eWeLink] [btm.tv.usb] sending update failed as Cannot read property 'updateCharacteristic' of undefined.
[04/01/2021, 17:53:16] [eWeLink] [btm.tv.usb] receiving update {"switches":[{"switch":"off","outlet":0},{"switch":"off","outlet":1},{"switch":"off","outlet":2},{"switch":"off","outlet":3}],"online":true,"updateSource":"WS"}.
[04/01/2021, 17:53:16] [eWeLink] [btm.tv.usb] sending update failed as Cannot read property 'updateCharacteristic' of undefined.
[04/01/2021, 17:53:16] [eWeLink] [btm.tv.usb] receiving update {"switches":[{"switch":"off","outlet":0},{"switch":"off","outlet":1},{"switch":"off","outlet":2},{"switch":"off","outlet":3}],"online":true,"updateSource":"LAN"}.
[04/01/2021, 17:53:16] [eWeLink] [btm.tv.usb] sending update failed as Cannot read property 'updateCharacteristic' of undefined.
```

**Comment by bwp91** [comment]
Hi @daiiig 
Thanks for reporting this.
This should be fixed in v4.1.0.
Thanks

**Comment by daiiig** [comment]
updated to latest version and no longer getting this error.  Thanks for quick response and resolution.
